content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -97,6 +153,8 @@ To encrypt the data:
 
 1. Create a new encryption configuration file using the appropriate properties for the `kms` provider:
 
+{{< tabs name="tab_with_md" >}}
```

Comment:
For this section, I would not go with a tab approach especially regarding the nice example in v2 where both can co-exist while the importance if the list order. 
In the current writing, it implies that v1 example can't have v2 in while v2 would tolerate v1 in.

Reply:
Updated the example for v1, but leaving the tabs since `apiVersion` is new in v2.